service/controller/app/v1/resource/chartoperator/resource.go

```diff
@@ -168,9 +168,16 @@ func (r Resource) installChartOperator(ctx context.Context, cr v1alpha1.App) err
 	}
 
 	{
-		err = cc.HelmClient.InstallReleaseFromTarball(ctx, tarballPath, namespace, helm.ReleaseName(release), helm.ValueOverrides(chartOperatorValues))
-		if err != nil {
-			return microerror.Mask(err)
+		if update {
+			err = cc.HelmClient.UpdateReleaseFromTarball(ctx, release, tarballPath, helm.UpdateValueOverrides(chartOperatorValues), helm.UpgradeForce(true))
```

Comment:
`helm.UpgradeForce(true)` added so we are not updating revision number and to ensure that release not fall into the loop.